app/javascript/components/App.js

```diff
@@ -26,8 +26,8 @@ class App extends Component {
   }
 
   componentDidMount(){
-    Promise.all(this.readNomaddaDestinations(),
-    this.readNomaddaCountries())
+    Promise.all([this.readNomaddaDestinations(),
```

Comment:
Does this work?

Reply:
yes that's the only way it works in an array and having them both. Kevin helped us find that out